### PR TITLE
DDF-2259 Fixed Registry IdentificationPlugin update method and reorde…

### PR DESCRIPTION
**Pull request by clockard**
#### What does this PR do?
Fixes a couple of registry bugs
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@mcalcote @gordocanchola @vinamartin 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@coyotesqrl
@shaundmorris

#### How should this be tested?
Build and start up two instances. Have one instance subscribe to the other. In the instance that subscribed to the other, look in solr and examine the metadata for the external identifiers. Make an update to the identity node on the other instance and after 30 seconds check the solr again and verify the ebrim external identifiers didn't change.
#### What are the relevant tickets?
DDF-2259
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

…red the handleEvent method of RegistryStorePublisher.

### CODE REVIEW

**Comment by shaundmorris** [comment]
👍

File: catalog/spatial/registry/registry-source-configuration-handler/src/main/java/org/codice/ddf/registry/source/configuration/SourceConfigurationHandler.java [hunk 1]
```diff
@@ -228,8 +228,7 @@ private synchronized void updateRegistryConfigurations(Metacard metacard, boolea
         for (ServiceBindingType bindingType : bindingTypes) {
             Map<String, Object> slotMap = this.getServiceBindingProperties(bindingType);
 
-            String factoryPidMask = slotMap.get(BINDING_TYPE)
-                    .toString();
+            String factoryPidMask = (String)slotMap.get(BINDING_TYPE);
```
Comment (vinamartin) [inline catalog/spatial/registry/registry-source-configuration-handler/src/main/java/org/codice/ddf/registry/source/configuration/SourceConfigurationHandler.java]: :shipit:

**Comment by coyotesqrl** [comment]
👍

**Comment by shaundmorris** [comment]
The bamboo build failed on this PR because the master nightly snapshot failed. So there are a couple of artifacts that are outdated.

**Comment by stustison** [comment]
Lets merge this. We also need a 2.9.x PR

**Pull request merged**